### PR TITLE
chore: Clean up press events for articles

### PR DESCRIPTION
**Pull request by tuncaulubilge**
Standardized onPress events for article-bylines and related articles and added addon-actions for
storybook to prevent default actions and log the event

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/3dfc445be9f98661491a2c4ac77e29da6995012d/index.html

File: packages/article/related-articles/related-articles.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const RelatedArticles = ({ articles, mainId, onPress, template }) => {
         headlineClass={headlineClass}
         id={article.id}
         imageContainerClass={imageContainerClass}
-        onPress={() => onPress({ url: article.url })}
+        onPress={e => onPress(e, { url: article.url })}
```
Comment (colincclark) [inline packages/article/related-articles/related-articles.js]: can you add tests for these onPress events please?

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/f06cd895ab77c37b21ac42b15fc03c38dd68b1bb/index.html

File: packages/article/related-articles/related-articles.js (hunk 1 above)
Comment (tuncaulubilge) [reply packages/article/related-articles/related-articles.js]: Added in f06cd89

**Pull request merged**